src/routes/+page.svelte

```diff
@@ -1,2 +1,151 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://svelte.dev/docs/kit">svelte.dev/docs/kit</a> to read the documentation</p>
+<script>
+    import { Header, Flyout, Button, HeaderTypeEnum, Text, Popover, PopoverChipTrigger, PopoverMultiSelectContent } from "kiwi-nl";
+    import {presetStudentData} from "../presetStudentData"
+    import humanIcon from "$lib/img/human-icon.png"
+    let flyoutElement;
+
+</script>
+
+<main>
+  <Flyout bind:this={flyoutElement} header={presetStudentData.name} subheader="">
+    <div slot="flyout-body" class="flyout-body">
+      <Header type={HeaderTypeEnum.h3}>Academic Information: </Header>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Academic College: </Header>
+        <Text>{presetStudentData.college}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Major: </Header>
+        <Text>{presetStudentData.major}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Minor: </Header>
+        <Text>{presetStudentData.minor}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>GPA (Current): </Header>
+        <Text>{presetStudentData.GPA}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Current Semester Credit Hour Amount: </Header>
+        <Text>{presetStudentData.currentCreditHours}</Text> 
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Total Credits Taken: </Header>
+        <Text>{presetStudentData.creditsTaken}</Text>
+      </div>
+
+      <br> <Header type={HeaderTypeEnum.h3}>Advisor Information: </Header>
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Academic Advisor: </Header>
+        <Text>{presetStudentData.acadmicAdvisorName}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Academic Advisor Email: </Header>
+        <Text>{presetStudentData.academicAdvisorEmail}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Co-op Advisor: </Header>
+        <Text>{presetStudentData.coopAdvisorName}</Text>
+      </div>
+
+      <div class="group-elements">
+        <Header type={HeaderTypeEnum.subheader}>Co-op Advisor Email: </Header>
+        <Text>{presetStudentData.coopAdvisorEmail}</Text>
+      </div>
+      
+      <br> <Header type={HeaderTypeEnum.h3}>Progress Information: </Header>
+
+    </div>
+  </Flyout> 
+
+  <div class="header-card">
```

Comment:
I think we should put "My Academics" here as an overall header to the page, since everything covered in this page ("Classes and Schedule", "Graduation and Commencement", etc.) falls under that umbrella